backend/app/api/v1/endpoints/quay/quayGraphs.py

```diff
@@ -0,0 +1,319 @@
+    safe_add(data['aggregations'], resultData, 'status_codes.204', 'status_code_2XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.400', 'status_code_4XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.401', 'status_code_4XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.403', 'status_code_4XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.404', 'status_code_4XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.405', 'status_code_4XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.408', 'status_code_4XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.500', 'status_code_5XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.502', 'status_code_5XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.503', 'status_code_5XX')
+    safe_add(data['aggregations'], resultData, 'status_codes.504', 'status_code_5XX')
+    return resultData
+
+
+async def parseImageResults(data: dict):
+    totals = {'latency': 0.0, 'success_count': 0.0, 'failure_count': 0.0}
+    datapoints = data['aggregations']['uuid']['buckets']
+    for each in datapoints:
+        safe_add(each, totals, 'latency', 'latency')
+        safe_add(each, totals, 'success_count', 'success_count')
+        safe_add(each, totals, 'failure_count', 'failure_count')
+    totals['latency'] /= len(datapoints)
+    totals['success_count'] /= len(datapoints)
+    totals['failure_count'] /= len(datapoints)
+    return totals
+
+
+async def getImageMetrics(uuids: list, index: str):
+    ids = "\" OR uuid: \"".join(uuids)
+    print(ids)
+    query = {
+        "size": 0,
+        "aggs": {
+            "uuid": {
+            "terms": {
+                "field": "uuid.keyword"
+            },
+            "aggs": {
+                "latency": {
+                    "avg": {
+                        "field": "elapsed_time"
+                    }
+                },
+                "success_count": {
+                    "sum": {
+                        "field": "success_count"
+                    }
+                },
+                "failure_count": {
+                    "sum": {
+                        "field": "failure_count"
+                    }
+                }
+            }
+            }
+        },
+        "query": {
+            "bool": {
+                "must": [{
+                    "query_string": {
+                        "query": (
+                            f'( uuid: \"{ids}\" )'
+                        )
+                    }
+                }]
+            }
+        }
+    }
+    print(query)
+    es = ElasticService(configpath="quay.elasticsearch",index=index)
+    results = await es.post(query,size=0)
+    await es.close()
+    return results
+
+
+async def getQuayMetrics(uuids: list, index: str):
+    ids = "\" OR uuid: \"".join(uuids)
+    print(ids)
```

Comment:
We print the query and the ids here... can we just print one or the other? (the query should suffice?)

Reply:
Removed ids.